src/mantine-core/src/components/Collapse/demos/accessibility.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Button } from '../../Button/Button';
+import { Collapse } from '../Collapse';
+
+const code = `
+const [isOpen, setOpen] = useState(false);
+
+  return (
+    <div style={{ maxWidth: 400 }}>
+      <Button
+        onClick={() => setOpen(prev => !prev)}
+        aria-expanded={isOpen}
+        aria-controls="collapse"
+        id="collapse-button"
+      >
+        Collapse
+      </Button>
+
+      <Collapse in={isOpen} aria-labelledby="collapse-button" id="collapse">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
```

Comment:
Do not include large sections of the content in demo code, replace it with comment `{/* content */}`